_extras/snippets_library/UCL_sge/14/ls_dir.snip

```diff
@@ -1 +1 @@
-[yourUsername@cirrus-login0 ~]$ ls /lustre/sw/anaconda/anaconda3-5.1.0/bin
+[cceagil@login13 ~]ls /shared/ucl/apps/python/3.6.3/gnu-4.9.2/bin
```

Comment:
the `$` in the prompt should be there too.

Reply:
Will `{{ site.host_prompt }}` add it automatically?
We probably need to update the `_extras/snippets_library/UCL_sge/_config_options.yml` file.